Tests/Controller/ExceptionControllerTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+namespace SumoCoders\FrameworkErrorBundle\Tests\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Symfony\Component\DomCrawler\Crawler;
+use Symfony\Bundle\FrameworkBundle\Client;
+
+class DefaultControllerTest extends WebTestCase
+{
+    public function test404Page()
+    {
+        $client = static::createClient();
+        $crawler = $client->request('GET', '/non-existing-path');
+
+        $this->assertErrorPage($client, $crawler, 404, 'No route found for "GET /non-existing-path"');
```

Comment:
you really want the error to appear on the page? I think this could be dangerous some times! If you have a doctrine error, you really don't want SQL to appear.